docs/topics/conditional-view-processing.txt

```diff
@@ -64,7 +64,15 @@ order, as the view function they are helping to wrap. The function passed
 ``last_modified_func`` should return a standard datetime value specifying the
 last time the resource was modified, or ``None`` if the resource doesn't
 exist. The function passed to the ``etag`` decorator should return a string
-representing the `Etag`_ for the resource, or ``None`` if it doesn't exist.
+representing the `ETag`_ for the resource, or ``None`` if it doesn't exist.
+
+.. versionchanged:: 1.11
+
+    In prior versions the return value from ``etag_func()`` was interpreted as
+    the unquoted part of the ETag. That prevented the use of weak ETags, which
+    have the format ``W/"<string>"``. The return value is now expected to be
+    an ETag as defined by the specification (including the quotes), although
+    the unquoted format is also accepted for backwards compatibility reasons.
```

Comment:
Is it fine to remove this entire note when removing or should anything remain? If the latter, please make the note self-contained as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.